EIPS/eip-615.md

```diff
@@ -45,7 +45,7 @@ The result is that all of the following validations and optimizations can be don
 
 ### Dependencies
 
-> **[EIP-1702](https://github.com/ethereum/EIPs/pull/1702). Generalized Account Versioning Scheme.** This proposal needs a versioning scheme to allow for its bytecode (and eventually eWasm bytecode) to be deployed with existing bytecode on the same blockchain.
+> **[EIP-1702](http://eips.ethereum.org/EIPS/eip-1702). Generalized Account Versioning Scheme.** This proposal needs a versioning scheme to allow for its bytecode (and eventually eWasm bytecode) to be deployed with existing bytecode on the same blockchain.
```

Comment:
Btw if this is a dependency, it should have been added to the `requires` header.